engine/lib/phx/src/engine/event_bus.rs

```diff
@@ -0,0 +1,303 @@
+use mlua::{Function, Lua, Table};
+use std::{
+    borrow::BorrowMut,
+    cell::Ref,
+    collections::{hash_map::Entry, BinaryHeap, HashMap},
+    sync::atomic::{AtomicU32, Ordering},
+};
+use strum::IntoEnumIterator;
+use strum_macros::EnumIter;
+use tracing::trace;
+
+use internal::ConvertIntoString;
+
+#[luajit_ffi_gen::luajit_ffi]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, EnumIter)]
+pub enum UpdatePass {
+    PreSim,             // Before physics update
```

Comment:
If you put these comments before the enum values, like this:
```suggestion
    /// Before physics update
    PreSim,
```
I think they will get incorporated into luajit-ffi-gen, at least eventually